### ISSUE TITLE
An error sometimes pop up when close Gd launcher

### ISSUE THREAD

**Issue by Jordan231111** [killpowa/Twickt-Launcher#1077]
**Describe the bug**
Error when closing application
A fail fast exception occured exception handlers will not be invoked and the process will be terminated immediately
**To Reproduce**
1. Go to Open the application GD launcher
2. do whatever u wanna do
3. close application
4. Sometimes when closing GD launcher this error come out
**5. GDLauncher.exe - Fail Fast Exception
![csrss_mfqqSPGGTt](https://user-images.githubusercontent.com/79342877/136115910-74986ff3-bb09-45eb-9f06-79ece35a610b.png)
A fail fast exception occurred. Exception handlers will not be invoked and the process will be terminated immediately.**

**Expected behavior**
**The application should close without fail fast exception**

Some ideas of how you would solve this issue, if possible
perhaps find a way to not make application not give out any error when closing application in any way (dont need to be high priority though).

windows 10 

**Additional context**

**Comment by Hraefin** [comment]
---------------------------
GDLauncher.exe - Fail Fast Exception
---------------------------
A fail fast exception occurred. Exception handlers will not be invoked and the process will be terminated immediately. 
---------------------------
OK   
---------------------------

this is the dialog box windows pops up on closing GDLauncher.

**Comment by svr333** [comment]
I have the exact same issue, though it is consistent and I get it every time I close the launcher

**Comment by svr333** [comment]
When running through cmd, this is the following output once I close down the application
![image](https://user-images.githubusercontent.com/35664724/149654521-28191eae-e262-434a-bf11-a7162ea9f876.png)

**Comment by Dr69onS6mur6i** [comment]
I've been getting this consistently for the past week or so. Every time I close GDLauncher I get this same error. Even without launching a game.

**Comment by svr333** [comment]
Yeah I also still get this error every time I shut down GDLauncher
